server/node-server.js

```diff
@@ -34,6 +36,9 @@ passport.use(new FacebookStrategy({
 // Facebook will redirect the user back to the application at
 //     /auth/facebook/callback
 app.get('/auth/facebook', passport.authenticate('facebook'));
+app.get('/me', checkAuthorizationToken(), authenticate(), (request, response) => {
+  response.json(request.user);
+});
```

Comment:
Does this take care of the initial login step? Like if the user isn't logged in how is it dealing with the Facebook redirect?